Jenkinsfile

```diff
@@ -15,6 +15,7 @@ node('rhel8'){
 		sh "curl -Ls https://sh.jbang.dev | bash -s - app setup"
 		env.PATH="~/.jbang/bin:${env.PATH}"
 		sh "jbang trust add https://github.com/apache/"
+		//add JQ and ciclonedx here? maybe they are already installed as tools?

```

Comment:
yes, in case they are not already availabl eon the base image, this will come here, either by using a `tool` if provided as such, or calling the installation command.

Reply:
after trying on the CI, jq is installed, only need to install cyclonedx-npm

Reply:
Added cyclonedx and cyclonedx-npm installation to the 'install requirements' stage.